style/src/slider.rs

```diff
@@ -5,11 +5,20 @@ use iced_core::Color;
 #[derive(Debug, Clone, Copy)]
 pub struct Appearance {
     /// The colors of the rail of the slider.
-    pub rail_colors: (Color, Color),
+    pub rail: Rail,
     /// The appearance of the [`Handle`] of the slider.
     pub handle: Handle,
 }
 
+/// The appearance of a slider rail
+#[derive(Debug, Clone, Copy)]
+pub struct Rail {
+    /// The colors of the rail of the slider.
+    pub rail_colors: (Color, Color),
+    /// The height of a slider rail and the width of a vertical slider.
+    pub rail_size: f32,
```

Comment:
Now we are doing this we should add `border_radius`, and maybe `border_color` as well as `border_width`. Then we can get come nice rounded sliders.

<img width="288" alt="Screenshot 2023-01-26 at 9 58 00 AM" src="https://user-images.githubusercontent.com/2248455/214796000-fac7cc21-ae33-4ce8-b9f8-2c4425b8ea92.png">